### ISSUE TITLE
Checkbox and Functions to turn off token editing and movement.

### ISSUE THREAD

**Issue by aliasmask** [RPTools/maptool#1762]
**Is your feature request related to a problem? Please describe.**
Sometimes you want players to interact with a token (run a macro on token, select the token, see the token) but not be able to change anything about the token (no right click menu, no movement, no double click to edit token) that players can own.

**Describe the solution you'd like**
Add a checkbox to the config panel to turn off player editing and another one to turn off player movement. The GM will still have permission when these are set.

**Additional context**
I've seen this request before in discord and I think the forums. It would be mostly for unruly players. But I've thought of another good use case. The GM can create tokens that can be seen by only certain players but not interacted with. This would be good for invisible tokens. One issue with creating a situation where a token can be seen by some players and not others is you either have to put the token on the object layer where it will always be seen and ignores vision rules or on the token layer where it can be interacted with by the players. This solves both those issues and allows for more GM control over tokens on the board. Win/Win.

Players could still run macros on or against these tokens, select them, view the statsheet as if owning the token. Only the editing and movement would be impacted limiting the amount of code changes required.

**Comment by aliasmask** [comment]
Is this related to #975? I downloaded 1.7.0 beta-1 and I don't see how to implement.

**Comment by Phergus** [comment]
It's related in that the feature added in #975 allows the GM to lock out players from editing tokens.

**Tool menu -> Lock Player Token Editor**

**Comment by aliasmask** [comment]
Yeah, it would be nice if this could be token specific when gm shares a token with player.
